src/messaging/ReliableMessageProtocolConfig.cpp

```diff
@@ -126,7 +126,8 @@ Optional<ReliableMessageProtocolConfig> GetLocalMRPConfig()
 }
 
 System::Clock::Timeout GetRetransmissionTimeout(System::Clock::Timeout activeInterval, System::Clock::Timeout idleInterval,
-                                                System::Clock::Timeout lastActivityTime, System::Clock::Timeout activityThreshold)
+                                                System::Clock::Timeout lastActivityTime, System::Clock::Timeout activityThreshold,
+                                                bool isPeerActive)
```

Comment:
So... maybe if we want to match `ReliableMessageMgr::CalculateNextRetransTime` we should just explicitly have a isResponseMessage boolean argument here?  That would make it _very_ clear what is actually going on and how to go about deciding what to pass in.

Reply:
I am now using isInitial boolean, which should be similar to !isResponseMessage, thanks